tests/Library/Vanilla/Utility/HtmlSnippetTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @copyright 2009-2020 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\Library\Vanilla\Utility;
+
+use PHPUnit\Framework\TestCase;
+use Vanilla\Utility\HtmlSnippet;
+
+/**
+ * Class HtmlSnippetTest
+ *
+ * @package VanillaTests\Library\Vanilla\Utility
+ */
+class HtmlSnippetTest extends TestCase {
+
+    /**
+     * @var HtmlSnippet
+     */
+    private $html;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp(): void {
+        parent::setUp(); // TODO: Change the autogenerated stub
+        $this->html = new HtmlSnippet();
+    }
+
+    /**
+     * Test Dom saveXML method.
+     */
+    public function testBasicSaveXml(): void {
+        $html = 'Hello World!.';
+        $this->html->loadHtml($html);
+        $result = $this->html->saveXML();
+        $this->assertSame($html, substr($result, 22, -6));
+    }
+
+    /**
+     * Test Dom saveHtml method.
+     */
+    public function testBasicSaveHtml(): void {
+        $html = <<<HTML
+<b>Hello World!.🤓</b>
+HTML;
+        $this->html->loadHtml($html);
+        $result = $this->html->saveHtml();
+        $this->assertSame($html, substr($result, 22, -6));
```

Comment:
This smells. You don't change the test. You fix the code so the test passes as it should.